dynawo/sources/Models/Modelica/Dynawo/Examples/Nordic/Components/GeneratorWithControl/GeneratorSynchronousFourWindingsWithControl.mo

```diff
@@ -0,0 +1,25 @@
+within Dynawo.Examples.Nordic.Components.GeneratorWithControl;
+
+  /*
+  * Copyright (c) 2022, RTE (http://www.rte-france.com)
+  * See AUTHORS.txt
+  * All rights reserved.
+  * This Source Code Form is subject to the terms of the Mozilla Public
+  * License, v. 2.0. If a copy of the MPL was not distributed with this
+  * file, you can obtain one at http://mozilla.org/MPL/2.0/.
+  * SPDX-License-Identifier: MPL-2.0
+  *
+  * This file is part of Dynawo, an hybrid C++/Modelica open source time domain simulation tool for power systems.
+  */
+
+model GeneratorSynchronousFourWindingsWithControl
+  
+  // Imports
```

Comment:
Not needed

Reply:
[d146726](https://github.com/dynawo/dynawo/pull/2394/commits/d14672683da0d677556a70ce6fab3936ca672551)